### ISSUE TITLE
[General Issue] Stop training by loss value 

### ISSUE THREAD

**Issue by Yasminbr** [PaddlePaddle/PaddleSeg#2292]
Hey , 
I would like o know if there is an option to stop training the model not by iteration numbers , but by seeing if whether the learning rate or the loss are static or the delta between some values is smaller than a number .

thank you :)

**Comment by wuyefeilin** [comment]
You can add some situation to the training circulation， and break out of the loop when the condition is met.

**Comment by Yasminbr** [comment]
that's sounds great, can you show me a code example?
